### DISCUSSION TITLE
当我小程序消息推送后，按照手册处理message

### DISCUSSION THREAD

**Discussion by b527847300** [w7corp/easywechat#2782] [Q&A]
当我小程序消息推送后，按照手册处理message。
$message = $server->getDecryptedMessage();
使用这个方法后返回的数据还都是未解密的
![Uploading image.png…]()

**Comment by b527847300** [comment]
![4079421587](https://github.com/w7corp/easywechat/assets/54353219/6a1bcf27-cf99-44d2-b5aa-eb6cd3db47ef)

**Comment by b527847300** [comment]
@overtrue 
大佬看一下，这是我使用的版本
 "w7corp/easywechat": "^6.8",

**Reply by tmkdahko** [reply]
![image](https://github.com/w7corp/easywechat/assets/3908082/e383cdbd-27b0-41f7-83c9-a22ab0b90854)
他这里有个问题，解密后当成xml转数组了，如果你需要把消息类型设置为xml格式才行。
